src/app/admin/manage-products/manage-products.component.ts

```diff
@@ -23,6 +23,7 @@ export class ManageProductsComponent implements OnInit {
   ) {}
 
   ngOnInit(): void {
+    localStorage.setItem('authorizationToken', 'Basic dmlzaG51MTMwMjpURVNUX1BBU1NXT1JE')
```

Comment:
No need to add Basic, it is done automatically by the service

Reply:
I did not understand how it was done by service. Can you explain more

Reply:
Also, Angular app was designed like that and given. I did not change any much code in it related to calling services. I should check why it's triggering twice and routing will be blocked to admin/products I think. This was not changed by me anyway. I am using the same app provided to me for course purposes. I see the products are getting uploaded properly. Attaching my csv file here[
[products.csv](https://github.com/Vishnu1302/shop-angular-cloudfront/files/11072183/products.csv)
](url)

Reply:
I mean no need to add the word Basic to your token, this will be added automatically by authorizer which you define here:
![image](https://user-images.githubusercontent.com/28926420/227790398-b186c145-6076-45f4-b55e-99ade248965d.png)

![image](https://user-images.githubusercontent.com/28926420/227790367-2fa1f884-1a16-4cfd-826e-5745adb46bf5.png)

https://goldspecdigital.com/blog/implementing-basic-auth-with-the-serverless-framework
https://developer.mozilla.org/en-US/docs/Web/HTTP/Authentication

Reply:
My bad about local storage item, I messed it up with React project where we add Basic to header  along with token:

![image](https://user-images.githubusercontent.com/28926420/227792850-fd9625d9-a385-4b0d-86b6-668ea303e0e8.png)

And in Angular it is a bit different, my apologize